.appveyor.yml

```diff
@@ -6,15 +6,17 @@ branches:
     - /^(.*\/)?ci-.*$/
     - /^rel\/.*/
 configuration:
-  - CodeAnalysis
+  - Debug
   - Release
```

Comment:
Thoughts? I'm not sure we bother building more than one `$(Configuration)` elsewhere.